src/main/rx/nrf24_kn.c

```diff
@@ -86,27 +86,27 @@ static void switch_channel(void)
 
 static void decode_bind_packet(uint8_t *packet)
 {
```

Comment:
Please don't include (unrelated) whitespace changes, it makes history and review complicated. Separate PR is necessary